ExtractCopyright.Tests/ExtractCopyrightTests.cs

```diff
@@ -178,7 +178,7 @@ public void CreateCopyrightFileWorks()
 				copyrights.AddOrUpdateParagraphFromAcknowledgement(ackDict[key], "");
 			}
 
-			Assert.LessOrEqual(ackDict.Keys.Count + 3, copyrights.Paragraphs.Count, "We should have the two standard paragraphs, one paragraph per acknowledgement, and at least one license paragraph");
+			Assert.LessOrEqual(ackDict.Keys.Count - 1 + 3, copyrights.Paragraphs.Count, "We should have the two standard paragraphs, one paragraph per acknowledgement, and at least one license paragraph");
```

Comment:
This extra magic number needs some explanation. The reason isn't clear. Perhaps it shouldn't have used any magic numbers to begin with, but this isn't helping.